src/workerd/api/tests/abortsignal-test.js

```diff
@@ -1,8 +1,52 @@
-import { strictEqual, ok, throws } from 'node:assert';
+import { strictEqual, ok, throws, strict } from 'node:assert';
+import { WorkerEntrypoint } from 'cloudflare:workers';
+import { scheduler } from 'node:timers/promises';
```

Comment:
Let's add a test that tries caching a local abort signal in the global scope in one request then tries triggering it in a different request. It should fail with an error and shouldn't crash anything.

```
let ac;

export default {
  fetch(req, env) {
    if (ac === undefined) {
      ac = new AbortController();
      env.FOO.rpcCall(ac.signal);  // send the signal over
    } else {
      ac.abort(new Error('boom?'));
    }
  }
}
```

The `ac.abort()` should fail with an error complaining about cross request I/O. Let's just double check that nothing weird happens with the rpc clients.